doc/en/user/source/data/raster/imagemosaic/configuration.rst

```diff
@@ -206,35 +206,35 @@ In addition to the required envelope and location attributes, the schema for the
      - N
      - Path to an auxiliary file to be used for internal purposes (For example: when dealing with NetCDF granules, it refers to the NetCDF XML ancillary file.)
    * - AbsolutePath
-     - Y
-     - Controls whether or not the path stored inside the ``location`` attribute represents an absolute path or a path relative to the location of the shapefile index. Notice that a relative index ensures much more portability of the mosaic itself. Default value for this parameter is ``false``, which means relative paths.
+     - N
```

Comment:
Similar to the previous comment, either this option is not actually required, or the example configuration shown below should include it (I think).